### PR TITLE
LST-23 Setup Minimal Node js

### LINKED ISSUE #23: Setup backend server
Setup a simple nodejs backend server 
something like: 
https://dev.to/bam92/how-to-setup-a-minimal-node-js-server-5479
https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/development_environment

### PR DESCRIPTION
**Pull request by nurbekismagulov**
This PR sets up Minimal node js server using express. Morgan is used as middleware to log https requests.
Closes #23

### CODE REVIEW

File: packages/backend/package.json [hunk 1]
```diff
@@ -2,18 +2,24 @@
   "name": "@lostie/backend",
   "version": "0.0.0",
   "license": "MIT",
-  "main": "index.js",
+  "main": "./dist/index.js",
   "scripts": {
     "build": "rimraf ./dist && tsc",
     "dev": "nodemon",
-    "start": "yarn build && node dist/server.js",
+    "start": "yarn build && node .",
```
Comment (nurbekismagulov) [inline packages/backend/package.json:9]: `node .` refers to "main" property in package.json, which is ./dist/index.js

File: packages/backend/src/shared/infra/api/v1/router.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+// import {
+//   adminRouter,
+//   clientRouter,
+// } from '../../../../modules/users/infra/http/routes';
+// import {
+//   bookingRouter,
+//   paymentRouter,
+//   pitchRouter,
+//   placeRouter,
+// } from '../../../../modules/booking/infra/http/routes';
```
Comment (oizuldan) [inline packages/backend/src/shared/infra/api/v1/router.ts:11]: do we really need this and the below ones?

File: packages/backend/src/shared/infra/api/v1/router.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+const v1Router = express.Router();
+
+v1Router.get('/', (_, res) => {
+  return res.json({ message: "api/v1/ Yo! we're up" });
```
Comment (oizuldan) [inline packages/backend/src/shared/infra/api/v1/router.ts:16]: the difference between 
```
app.get('/', (_, res) => {
  return res.json({ message: "Yo! we're up" });
});
```
and this is routing only right?

**Review by oizuldan — APPROVED** [review]
lgtm!

File: packages/backend/src/shared/infra/api/v1/router.ts (hunk 2 above)
Comment (nurbekismagulov) [reply packages/backend/src/shared/infra/api/v1/router.ts:11]: I left these comments as an example of how to include different routes, after we'll create them

File: packages/backend/src/shared/infra/api/v1/router.ts (hunk 3 above)
Comment (nurbekismagulov) [reply packages/backend/src/shared/infra/api/v1/router.ts:16]: https://stackoverflow.com/questions/28305120/differences-between-express-router-and-app-get - here is the explanation of how they differs

**Pull request merged**